tests/test-umockdev-record.vala

```diff
@@ -165,23 +165,6 @@ A: knobs/red=off
 }
 
 
-// tracing ioctls does not work in testbed
-static void
-t_testbed_no_ioctl_record ()
-{
-    string sout;
-    string serr;
-    int exit;
-
-    var tb = new UMockdev.Testbed ();
-    tb.add_devicev ("mem", "zero", null, {"dev", "1:5"}, {});
-    spawn ("umockdev-record" + " --ioctl /sys/devices/zero=/dev/stdout -- " + readbyte_path + " /dev/zero",
-           out sout, out serr, out exit);
-    Assertions.assert_cmpint (exit, Op.NE, 0);
-    Assertions.assert_cmpstr (sout, Op.EQ, "");
-    assert (serr.contains ("UMOCKDEV_DIR cannot be used"));
```

Comment:
Please remove the corresponding check from the preload as well then. I actually wonder why that even works, as umockdev-record now creates/sets that env, so why doesn't ioctl_record_open() barf on this?